content/tutorials/dsx-subscribe.textile

```diff
@@ -0,0 +1,249 @@
+
+<a href="/images/tutorials/dsx-dashboard.png" target="_blank">
+  <img src="/images/tutorials/dsx-dashboard.png" style="width: 100%" alt="DSX Dashboard">
+</a>
+
+To confirm the subscription to the said data stream, switch to the 'Subscriptions' tab, and you should see the data stream listed there. You can also see the message quota, license type and other information related to your subscription to the said data stream.
+
+<a href="/images/tutorials/subscriptions-tab-dsx.png" target="_blank">
+  <img src="/images/tutorials/subscriptions-tab-dsx.png" style="width: 100%" alt="DSX Dashboard">
+</a>
+
+Now that your account has a valid subscription with the product let's go ahead and write some code to start receiving updates.
+
+h2. Step 3 - Buiding the HTML to display results
+
+In this tutorial, we'll be subscribing to arrival updates for London tube lines at certain stations, that you can select from a drop-down menu. Go ahead and paste the following basic HTML in a file and name it @index.html@. Note that each of the options in the drop-down menu corresponds to the respective Naptan Code as illustrated in the "Ably-TFL product documentation":https://dsx-sources.herokuapp.com/source/tfl. The Naptan Code is a combination of @line@ and @Station ID@. Note that we've also added a reference to Ably's JavaScript library, the jQuery library and a @main.js@ file in which we'll include our app logic.
+
+```[html]
+<html>
+<head>
+  <title>DSX subscriber tutorial</title>
+  <script src="https://cdn.ably.io/lib/ably.min-1.js" crossorigin="anonymous"></script>
+  <script
+  src="https://code.jquery.com/jquery-3.3.1.min.js"
+  integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
+  crossorigin="anonymous"></script>
+</head>
+<body>
+  <div>
+    <div>
+      Station:
+      <select id="naptan-code">
+        <option value="northern:940GZZLUEUS">Euston Underground Station</option>
+        <option value="northern:940GZZLUKSX">Kings Cross Underground Station</option>
+        <option value="jubilee:940GZZLUWSM">Westminster Underground Station</option>
+        <option value="northern:940GZZLUWLO">Waterloo Underground Station</option>
+      </select>
+    </div>
+  </div>
+  <table id="table-tfl" class="c-flashbox" style="margin-top: 20px;">
+    <thead>
+      <tr>
+        <th>Due</th>
+        <th>Destination</th>
+        <th>Platform</th>
+        <th>Line</th>
+      </tr>
+    </thead>
+    <tbody>
+    </tbody>
+  </table>
+<script src="main.js"></script>
+</body>
+</html>
+```
+
+h2. Step 4 - Determining the channel names
+
+In DSX, the channels available in any product are be determined by the naming convention described below:
+
+[product:<strong>DSX-ID</strong>]:<strong>NAMESPACE</strong>:<strong>OPTIONAL-CATEGORY</strong>
```

Comment:
We should probably describe what these things mean + provide links to deeper explanations.